src/config/dynamic-theme-fixes.config

```diff
@@ -7482,7 +7482,20 @@ div.sumo-nav--logo
 svt.se
 
 INVERT
-.nyh_navigation__nyh_logo-img
+header img[src*=svt]
+.nyh_navigation__menu-toggle
+.VideoPlayerTheme__play-pause-button-simple___vsG6v::before
+
+CSS
+.VideoPlayerTheme__live__splash___LA0AL, .nyh_breaking__top-prefix {
```

Comment:
```CSS
.VideoPlayerTheme__live__splash___LA0AL
```
Can be replaced by
```CSS
[class^="VideoPlayerTheme__live__splash"]
```
and the please use a new line for the next selector after `,`.